Lesson-7/lesson-7-new/index.html

```diff
@@ -55,46 +55,45 @@ <h1 class="headline">What do we offer</h1>
 <p class="text_like">Real guest reviews</p>
 </div>
 </div>
-</article>
-<article class="last">
+</section>
+<section class="last">
 </div>
 <div class="heading_logo">
-    <img src="image/logo_vector.png" alt="heading_logo">
+    <img class="heading_logo_last" src="image/logo_vector.png" alt="heading_logo">
 </div>
 <div class="container">
-<link rel="stylesheet" href="footer.css">
 <div class="container_about">
     <ul>
     <li class="heading_about">About</li>
-    <li>Now Triphouse works</li>
-    <li>Careers</li>
-    <li>Privacy</li>
-    <li>Terms</li>
+    <li class="list_items">Now Triphouse works</li>
+    <li class="list_items">Careers</li>
+    <li class="list_items">Privacy</li>
+    <li class="list_items">Terms</li>
 </ul>
 </div>
 <div class="container_property_types">
     <ul>
     <li class="heading_property">Property types</li>
-    <li>Guest houses</li>
-    <li>Hotels</li>
-    <li>Apartments</li>
-    <li>Villas</li>
-    <li>Holiday homes</li>
-    <li>Hostels</li>
+    <li class="list_items">Guest houses</li>
+    <li class="list_items">Hotels</li>
+    <li class="list_items">Apartments</li>
+    <li class="list_items">Villas</li>
+    <li class="list_items">Holiday homes</li>
+    <li class="list_items">Hostels</li>
 </ul>
 </div>
 <div class="container_support">
     <ul>
     <li class="heading_support">Support</li>
-    <li>Contact Customer Service</li>
-    <li>FAQ</li>
+    <li class="list_items">Contact Customer Service</li>
+    <li class="list_items">FAQ</li>
 </ul>
 </div>
 </div>
 <footer>
     <p class="caption">&copy; 2020 Triphouse, lnc. All right reserved</p>
```

Comment:
check css. this tag is still the reason of horizontal scroll
![image](https://user-images.githubusercontent.com/20189313/129095842-be311313-62e1-4cc6-a5c3-619179971829.png)